cli/pcluster/config/validators.py

```diff
@@ -1253,3 +1253,25 @@ def fsx_ignored_parameters_validator(section_key, section_label, pcluster_config
 def _describe_ec2_key_pair(key_pair_name):
     """Return information about the provided ec2 key pair."""
     return boto3.client("ec2").describe_key_pairs(KeyNames=[key_pair_name])
+
+
+def ebs_volume_iops_validator(section_key, section_label, pcluster_config):
+    errors = []
+    warnings = []
+
+    section = pcluster_config.get_section("ebs", section_label)
```

Comment:
use `section_key` instead of `"ebs"` here

Reply:
Done.